docs/articles/licensing/publishing-binaries.rst

```diff
@@ -0,0 +1,30 @@
+Publishing binary images
+========================== 
+
+We release binary images of our products that are licensed under the 
```

Comment:
In the Blueprint context we are not releasing anything.  In this part of the SWF we should write about the effects certain conditions and actions have in a general sense. The different Deployments (e.g. PELUX Baseline) should go into details about the concrete effects licenses have on the project, e.g. how it affects the releases or CI&CD jobs.

Reply:
On the other hand, this sentence can easily be adapted to describe what happens when we do something considered distribution by the GPL. "When we release...".

However, it would be nice to have this be more general description of how licenses affect what we need to do when doing binary releases.